cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,34 @@
 /// <reference types='cypress' />
 
+
 describe('Book Store app', () => {
-  before(() => {
-    
+
+  it('login', () => {
+    cy.login();
+    cy.visit('/profile');
+    cy.get('#userName-value').should('contain', 'serhio1234');
+    cy.url().should('include', '/profile');
+  });
+
+  it('add book', () => {
```

Comment:
Use a more descriptive test name, like 'should add a book to the collection'. It helps to understand what the test is doing and what to expect from it.

Reply:
Changed